tests/existing-resources/README.md

```diff
@@ -0,0 +1,3 @@
+# Existing Resources For use in tests
+
+<!-- Add an example where existing resources can be passed as variables for local testing (not for use with tests) -->
```

Comment:
`not for use with tests` is confusing - we are using it for tests, but as a prereq deployment to test the existing resources example